proxies/shared/policies/AssignMessage.Messages.GetSingle.Request.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+ This policy populates the requestpath, Idempotency and Correlation ID Headers, and the body of the request before it reaches the backend.
+
+ Since the payload is JSON we can't use the standard {variable.name} syntax for Apigee flow variables, this is where we use the variablePrefix and variableSuffix attributes.
+
+ For more information see the following resources:
+    * https://docs.apigee.com/api-platform/reference/policies/assign-message-policy#set-payload
+-->
+<AssignMessage async="false" continueOnError="false" enabled="true" name="AssignMessage.Messages.GetSingle.Request">
+    <DisplayName>AssignMessage.Messages.GetSingle.Request</DisplayName>
+    <Properties/>
+    <AssignTo createNew="false" transport="http" type="request"/>
+    <IgnoreUnresolvedVariables>true</IgnoreUnresolvedVariables>
+    <AssignVariable>
+        <Name>target.copy.pathsuffix</Name>
+        <Value>false</Value>
+    </AssignVariable>
+    <AssignVariable>
+        <Name>requestpath</Name>
+        <Value>/api/v1/messages/{messageId}</Value>
```

Comment:
Is this correct? Shouldn't it be `data.messageId` as the extract variable has a prefix of `data` on it?

Reply:
Have merged 1897 which fixes this